src/main/java/org/phyloref/jphyloref/commands/TestCommand.java

```diff
@@ -123,12 +143,18 @@ public void execute(CommandLine cmdLine) throws RuntimeException {
         // Ontology loaded.
         System.err.println("Loaded ontology: " + ontology);
 
-        // Reason over the loaded ontology.
+        // Reason over the loaded ontology -- but only if the user wants that!
         JFactReasonerConfiguration config = new JFactReasonerConfiguration();
-        JFactReasoner reasoner = new JFactReasoner(ontology, config, BufferingMode.BUFFERING);
+        JFactReasoner reasoner = null;
+        Set<OWLNamedIndividual> phylorefs;
 
         // Get a list of all phyloreferences.
-        Set<OWLNamedIndividual> phylorefs = PhylorefHelper.getPhyloreferences(ontology, reasoner);
+        if(flag_no_reasoner) {
+            phylorefs = PhylorefHelper.getPhyloreferences(ontology);
+        } else {
+            reasoner = new JFactReasoner(ontology, config, BufferingMode.BUFFERING);
+            phylorefs = PhylorefHelper.getPhyloreferences(ontology, reasoner);
+        }
```

Comment:
Seems like the more elegant and redundancy-avoiding way to write this would be to create an instance of a no-op reasoner, or, if `PhylorefHelper.getPhyloreferences()` tolerates that, pass `NULL` as the second argument, if `flag_no_reasoner` is true.